tests/conftest.py

```diff
@@ -1,36 +1,48 @@
-"""Tests suite for `platform_driver_agent`."""
-
-import json
-from pathlib import Path
-
-import pytest
-from volttron.platform_driver.agent import initialize_agent
-from volttrontesting import TestServer
-
-TESTS_DIR = Path(__file__).parent
-TMP_DIR = TESTS_DIR / "tmp"
-FIXTURES_DIR = TESTS_DIR / "fixtures"
+# -*- coding: utf-8 -*- {{{
+# vim: set fenc=utf-8 ft=python sw=4 ts=4 sts=4 et:
+#
+# Copyright 2020, Battelle Memorial Institute.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This material was prepared as an account of work sponsored by an agency of
+# the United States Government. Neither the United States Government nor the
+# United States Department of Energy, nor Battelle, nor any of their
+# employees, nor any jurisdiction or organization that has cooperated in the
+# development of these materials, makes any warranty, express or
+# implied, or assumes any legal liability or responsibility for the accuracy,
+# completeness, or usefulness or any information, apparatus, product,
+# software, or process disclosed, or represents that its use would not infringe
+# privately owned rights. Reference herein to any specific commercial product,
+# process, or service by trade name, trademark, manufacturer, or otherwise
+# does not necessarily constitute or imply its endorsement, recommendation, or
+# favoring by the United States Government or any agency thereof, or
+# Battelle Memorial Institute. The views and opinions of authors expressed
+# herein do not necessarily state or reflect those of the
+# United States Government or any agency thereof.
+#
+# PACIFIC NORTHWEST NATIONAL LABORATORY operated by
+# BATTELLE for the UNITED STATES DEPARTMENT OF ENERGY
+# under Contract DE-AC05-76RL01830
+# }}}
 """Configuration for the pytest test suite."""
 
+import pytest
 
```

Comment:
if you add the src directory to sys.path here then in your tests you can use from platform_driver.agent import xyz

Reply:
Are you suggesting adding something like the following:

```
from pathlib import Path
import sys
src_path = str(Path.joinpath(Path(__file__).parent.parent, "src"))
sys.path.append(src_path)
```

Reply:
I wouldn't blindly append it but yeah something like that see https://github.com/eclipse-volttron/volttron-core/blob/main/conftest.py for how I did this.

Though this repo conftest is at the root I like it better under tests...then you go up two dirs and down src and you add that to the path.